Assignment2/alvarogar4-fromRDFtoJSON-LD.jsonld

```diff
@@ -0,0 +1,45 @@
+{
+  "@context": {
+    "xsd": "http://www.w3.org/2001/XMLSchema#"
+  },
+  "@graph": [
+    {
+      "@id": "http://example.org/User10A",
+      "http://example.org/hasName": "Pedro"
```

Comment:
You can define a prefix in the @context and reuse its URI base to define each type and attribute property.